omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentBytesResource1_10.java

```diff
@@ -0,0 +1,87 @@
+package org.openmrs.module.attachments.rest;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Obs;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.attachments.AttachmentsConstants;
+import org.openmrs.module.attachments.AttachmentsContext;
+import org.openmrs.module.attachments.obs.AttachmentComplexData;
+import org.openmrs.module.attachments.obs.ValueComplex;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.response.GenericRestException;
+import org.openmrs.module.webservices.rest.web.response.IllegalRequestException;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.openmrs.obs.ComplexData;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.openmrs.module.attachments.AttachmentsContext.getContentFamily;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/attachment")
```

Comment:
Instead of `"/attachment"` you should use `"/" + AttachmentsConstants.ATTACHMENT_URI`.